src/listener/server-hardware.js

```diff
@@ -23,9 +23,12 @@ THE SOFTWARE.
 import Listener from './base-listener';
 import { buildD3Chart } from '../charting/chart';
 import { getLogicalInterconnectsMap } from '../ov-brain';
+import { getDeviceName } from '../middleware/utils/lexer';
```

Comment:
Is the function called getDeviceNameAndHyperLink now?  This import will have to be changed to match.